packages/components/src/components/radio-button-group/radio-button-group.tsx

```diff
@@ -47,7 +47,7 @@ export class RadioButtonGroup {
   render() {
     const helperTextId = `helper-message-${this.internalId}`;
     return (
-      <fieldset class="radio-button-group">
+      <fieldset part="fieldset" class="radio-button-group">
```

Comment:
`fieldset` is not wrong, but I would name this part `base`. It's probably not evident anymore but we've had a convention to name "base" the root element of the component, in this case:

- `.radio-button-group` → `base`
- `.radio-button-group__container` → `container`